src/styles/style.css

```diff
@@ -0,0 +1,285 @@
+*, *::after, *::before{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.nav{
+    display: flex;
+    /* position: fixed; */
+    height: 64px;
+    width: 100%;
+    background-color: #E6EBFE;
+    box-shadow: 2px 2px 2px  #4a54b1;
+    justify-content: space-between;
+    transition: 0.5s;
+    z-index: 1;
+}
+.logo{
+    font-size: 26px;
+    line-height: 64px;
+    padding-left: 18px;
+    background-image: linear-gradient(to right,#152B79 25%, #3250bd);
+    -webkit-background-clip: text;
+    -moz-background-clip: text;
+    background-clip: text;
+    color: transparent;
+}
+.nav ul{
+    display: flex;
+    padding-right: 18px;
+}
+.nav ul li{
+    list-style-type: none;
+    margin: 0 15px;
+    line-height: 64px;
+}
+.nav ul li a{
+    color: #3250BD;
+    text-decoration: none;
+    font-family: 'Lato', sans-serif;
+}
+li:hover{
+    border-bottom: 4px solid #405BBF;
+}
+#diffrent_color{
+    color: #7D94E8;
+}
+.main{
+    padding: 23px 26px 0 26px;
+}
+.square_primary {
+    width: 100%;
+    height: 172px;
+    background-color: #E5E5E5;
+    border-radius: 8px;
+    margin-bottom: 97px;
+}
+.title{
+    font-size: 20px;
+    padding-bottom: 14px;
+    font-family: 'Montserrat', sans-serif;
+}
+.title:after{
```

Comment:
Use `::` double colon for CSS pseudo elements. This notation was introduced in CSS3.